### PR TITLE
Ensure indices used in substring() are within length 

### PR DESCRIPTION
**Pull request by MrHadiSatrio**
Made in response to #20. This PR will safeguard indices being used in `substring()` calls not to exceed the string's own length.

### CODE REVIEW

File: library/src/main/java/com/github/badoualy/datepicker/MonthView.java [hunk 1]
```diff
@@ -325,12 +325,13 @@ void bind(int year, int month, boolean selected, boolean beforeSelection) {
             this.year = year;
             this.month = month;
 
-            String text = MONTHS[month].substring(0, 3).toUpperCase(Locale.US);
+            String text = MONTHS[month];
+            String truncatedText = text.substring(0, Math.min(text.length(), 3)).toUpperCase(Locale.US);
             if (yearDigitCount > 0) {
-                text += yearOnNewLine ? "\n" : " ";
-                text += year % (int) Math.pow(10, yearDigitCount);
+                truncatedText += yearOnNewLine ? "\n" : " ";
```
Comment (badoualy) [inline library/src/main/java/com/github/badoualy/datepicker/MonthView.java]: Feels weird to call it truncated text then concat some stuff.
Can you keep it in the same `text` variable?

Comment (ridsatrio) [reply library/src/main/java/com/github/badoualy/datepicker/MonthView.java]: Sure. But then `MONTHS[month]` would be CL called twice..

How about if we use `monthName` or just `name` instead and keep `text` as it is? How does that sound to you?

Comment (MrHadiSatrio) [reply library/src/main/java/com/github/badoualy/datepicker/MonthView.java]: Ouch sorry, replied from within my phone's Gmail and it seems like I still have my work account connected there.

(If you don't mind, would you be so kind to delete that comment for me? I'd really appreciate it. Thanks. 😃)

Comment (badoualy) [reply library/src/main/java/com/github/badoualy/datepicker/MonthView.java]: I was thinking of 

```java
String text = MONTHS[month];
text =  text.substring(0, Math.min(text.length(), 3)).toUpperCase(Locale.US);
```

But we can also introduce a new variable `monthName`

Comment (MrHadiSatrio) [reply library/src/main/java/com/github/badoualy/datepicker/MonthView.java]: Ah yes, that's actually better. Don't know why I didn't even think of that.

**Review by badoualy — APPROVED** [review]

**Pull request merged**